services-directions-models/src/main/java/com/mapbox/api/directions/v5/DirectionsCriteria.java

```diff
@@ -417,4 +482,26 @@ private DirectionsCriteria() {
   })
   public @interface ApproachesCriteria {
   }
+
+  /**
+   * Retention policy for the various payment methods.
```

Comment:
Why do we call it a retention policy? It thought it means something different in the context of JVM 🤔

Reply:
I think it's legacy from the previous version, that we copy-and-paste whenever `Def`s are used. Feel free to change it